pkg/scheduler/scheduler_test.go

```diff
@@ -38,7 +38,7 @@ func TestBuildDependencyGraph(t *testing.T) {
 
 	sched := New(c, nil, 0)
 
-	dg, err := sched.BuildDependencyGraph(interfaces.DependencyGraphOptions{})
+	dg, err := sched.BuildDependencyGraph(interfaces.DependencyGraphOptions{ReplicaCount: 1})
```

Comment:
0 is a valid number here. Any way, you're right, I will add such tests is dedicated commits latter. The change here was done in order to keep current test behaviour since with ReplicaCount 0 no replicas will be created and thus no resources will be created